server/tilesource/__init__.py

```diff
@@ -20,21 +20,42 @@
 # Deal with a bug where PEP257 crashes when parsing __all__
 # flake8: noqa
 
+import collections
+from girder.constants import TerminalColor
+from girder import logger
+
 from .base import TileSource, TileSourceException, \
     TileSourceAssetstoreException
 
-try:
-    from .test import TestTileSource
-except ImportError:
-    from .dummy import DummyTileSource as TestTileSource
-try:
-    from .tiff import TiffGirderTileSource
-except ImportError:
-    from .dummy import DummyTileSource as TiffGirderTileSource
-try:
-    from .svs import SVSGirderTileSource
-except ImportError:
-    from .dummy import DummyTileSource as SVSGirderTileSource
+AvailableTileSources = collections.OrderedDict()
+all = [TileSource, TileSourceException, TileSourceAssetstoreException,
+       AvailableTileSources]
+
+sourceList = [
+    {'moduleName': '.tiff', 'className': 'TiffGirderTileSource'},
```

Comment:
Eventually, I'd like to exclusively use MIME types rather than file extensions, but we need to make several other steps (improve Girder's handling of MIME, reach a consensus on what the MIME of an SVS should be) before this can effectively happen.